src/routes/profile.js

```diff
@@ -86,6 +108,22 @@ router.post('/mod/ajax/processavatarrequest', isModMiddleware, (req, res) => {
       } else if (req.body.decision === false || req.body.decision === 'false') {
         console.log(`search lower user: ${foundReq.forUsername.toLowerCase()}`);
 
+        const pattern = /pending_avatars\/.*$/;
+
+        await s3.rejectAvatarRefactorFilePath(
+          foundReq.resLink.match(pattern)[0],
+        );
+        await s3.rejectAvatarRefactorFilePath(
+          foundReq.spyLink.match(pattern)[0],
+        );
+
+        foundReq.resLink = 'deleted';
+        foundReq.spyLink = 'deleted';
+
+        foundReq.markModified('resLink');
+        foundReq.markModified('spyLink');
+        await foundReq.save();
```

Comment:
Can you help me actually delete the avatarRequest object after we're done? 
No need to keep it around :)